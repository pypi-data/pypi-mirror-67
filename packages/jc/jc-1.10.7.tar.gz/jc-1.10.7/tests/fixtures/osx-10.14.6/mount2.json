[{"filesystem": "/dev/disk1s1", "mount_point": "/", "options": ["apfs", "local", "journaled"]}, {"filesystem": "devfs", "mount_point": "/dev", "options": ["devfs", "local", "nobrowse"]}, {"filesystem": "/dev/disk1s4", "mount_point": "/private/var/vm", "options": ["apfs", "local", "noexec", "journaled", "noatime", "nobrowse"]}, {"filesystem": "map -hosts", "mount_point": "/net", "options": ["autofs", "nosuid", "automounted", "nobrowse"]}, {"filesystem": "map auto_home", "mount_point": "/home", "options": ["autofs", "automounted", "nobrowse"]}, {"filesystem": "//brazil@MyCloudEX2Ultra._afpovertcp._tcp.local/brazil", "mount_point": "/Volumes/brazil", "options": ["afpfs", "nodev", "nosuid", "mounted by kelly"]}, {"filesystem": "/dev/disk1s3", "mount_point": "/Volumes/Recovery", "options": ["apfs", "local", "journaled", "nobrowse"]}]
