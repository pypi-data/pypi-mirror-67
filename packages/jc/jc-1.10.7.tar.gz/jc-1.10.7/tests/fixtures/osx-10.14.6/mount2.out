/dev/disk1s1 on / (apfs, local, journaled)
devfs on /dev (devfs, local, nobrowse)
/dev/disk1s4 on /private/var/vm (apfs, local, noexec, journaled, noatime, nobrowse)
map -hosts on /net (autofs, nosuid, automounted, nobrowse)
map auto_home on /home (autofs, automounted, nobrowse)
//brazil@MyCloudEX2Ultra._afpovertcp._tcp.local/brazil on /Volumes/brazil (afpfs, nodev, nosuid, mounted by kelly)
/dev/disk1s3 on /Volumes/Recovery (apfs, local, journaled, nobrowse)
